src/cmr/concepts/concept.js

```diff
@@ -667,8 +668,24 @@ export default class Concept {
           return
         }
 
+        if (ummKey === 'revisionIds') {
```

Comment:
We can avoid this if statement altogether if we do something like this, since all concepts have a `meta.revision-id`:
```js
      const {
          'concept-id': conceptId,
          'revision-id': revisionId,
      } = meta
      
      // then in the loop...
      
       this.setItemValue(
            `${conceptId}-${revisionId}`,
            ummKey,
            keyValue
          )
 ```